internal/edge/command_trace_observer.go

```diff
@@ -32,6 +32,10 @@ var cmdTraceObserver = &cobra.Command{
 	provides visualization for the most actionable data so you can investigate and
 	solve issues faster.`,
 	Example: "newrelic edge trace-observer list --accountId <accountID>",
+	PreRun: func(cmd *cobra.Command, args []string) {
```

Comment:
Change to PersistentPreRun.  Validate with command metadata tests (same with others)